notify/receivers_test.go

```diff
@@ -35,20 +35,9 @@ import (
 	"github.com/grafana/alerting/receivers/wecom"
 )
 
-func TestInvalidReceiverError_Error(t *testing.T) {
-	e := InvalidReceiverError{
-		Receiver: &GrafanaReceiver{
-			Name: "test",
-			UID:  "uid",
-		},
-		Err: errors.New("this is an error"),
-	}
-	require.Equal(t, "the receiver is invalid: this is an error", e.Error())
-}
-
 func TestReceiverTimeoutError_Error(t *testing.T) {
-	e := ReceiverTimeoutError{
-		Receiver: &GrafanaReceiver{
+	e := IntegrationTimeoutError{
```

Comment:
Shouldn't this be something like

```suggestion
	e := InvalidIntegrationError{
```

Reply:
No sure I understand. Do you suggest renaming `ReceiverTimeoutError` to `InvalidIntegrationError`?

Reply:
I'm not sure what happened there, when I commented GitHub showed something else.